core/src/main/java/org/apache/spark/api/java/StorageLevels.java

```diff
@@ -36,6 +36,7 @@
   public static final StorageLevel MEMORY_AND_DISK_SER = create(true, true, false, false, 1);
   public static final StorageLevel MEMORY_AND_DISK_SER_2 = create(true, true, false, false, 2);
   public static final StorageLevel OFF_HEAP = create(true, true, true, false, 1);
+  public static final StorageLevel OFF_HEAP_ONLY_DESER = create(false, true, true, true, 1);
```

Comment:
Do we need to document this anywhere else or is this kind or private?
Seems reasonable to me.

Reply:
good point. Maybe I should remove it and the vendors should construct this storage level by themselves, if they have deserialized off-heap entries.

It's not a user-facing thing.